tests/it/test_projects/common_scargo_project/conanfile.py

```diff
@@ -1,22 +1,22 @@
-from conans import CMake, ConanFile, tools
+from conans import CMake, ConanFile, tools  # type: ignore[import]
 
 
-class CommonScargoProjectConan(ConanFile):
+class CommonScargoProjectConan(ConanFile):  # type: ignore[misc, no-any-unimported]
     name = "common_scargo_project_path"
     version = "0.1.0"
     settings = "os", "compiler", "build_type", "arch"
     description = "Project description."
     url = "www.hello-world.com"
     generators = "cmake_find_package", "cmake"
 
-    def package(self):
+    def package(self) -> None:
```

Comment:
These changes need to be in conanfile.j2 as well

Reply:
I can do them, but mypy will not check them.

Reply:
done